pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go

```diff
@@ -106,6 +176,22 @@ func (r *ReconcileSriovNetworkNodePolicy) Reconcile(request reconcile.Request) (
 	defaultPolicy := &sriovnetworkv1.SriovNetworkNodePolicy{}
 	err := r.client.Get(context.TODO(), types.NamespacedName{Name: DEFAULT_POLICY_NAME, Namespace: Namespace}, defaultPolicy)
 	if err != nil {
+		if errors.IsNotFound(err) {
+			// Default policy object not found, create it.
+			defaultPolicy.SetNamespace(Namespace)
```

Comment:
Is it possible that namespace from reconcile.request is not from `sriov-network-operator`?

Reply:
Currently, the operator is designed to only work in namespace `sriov-network-operator`.